course_inventory/published_date.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import time
 import json
+import requests
```

Comment:
Doesn't look like `time` or `requests` are used anymore.

Reply:
Removed it